src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -45,6 +45,102 @@ describe('AEM Forms OOTB Content Tests', () => {
         onboardingHdler.disable();
     });
 
+    // Validate rendering of Data Source properties page with partially filled OAuth details.
+    describe('DataSource Tests', () => {
+        let dataSources = [
+                {name : 'salesforce-cloud-config', title : 'Salesforce Cloud Config'},
+                {name : 'microsoft-dynamics-365-cloud-config', title : 'Microsoft Dynamics 365 Cloud Config'}
+            ],
+            basePath = 'mnt/overlay/fd/fdm/gui/components/admin/fdmcloudservice/properties.html',
+            confPath = '/conf/${appId}/settings/cloudconfigs/fdm',
+            verifyDataSource = (dataSource) => {
+                //Open datasource in editing mode
+                browser.url(`${config.aem.author.base_url}/${basePath}?item=${confPath}/${dataSource.name}`);
+
+                //Verify title value populated with correct value
+                expect($(selectors.editor.dataSourceEditor.title)).toBeDisplayed();
+                expect($(selectors.editor.dataSourceEditor.title).getValue()).toEqual(dataSource.title);
+
+                //click authentication settings tab
+                expect($(selectors.editor.dataSourceEditor.authenticationTab)).toBeDisplayed();
```

Comment:
These are generic test wrt the FDM editor (which is already being tested as part of production validation pipeline), I don't see any validation of content from these test. The test checking name of data source and its title seem to be the only test validating content.